doc/faq.rst

```diff
@@ -26,6 +26,11 @@ See :ref:`contributing`. Before wanting to add a new algorithm, which is
 usually a major and lengthy undertaking, it is recommended to start with :ref:`known
 issues <easy_issues>`.
 
+
+How can I make a bunch object?
+--------------------------------
+Don't make a bunch. They are not part of the scikit-learn API. They are just a way package some numpy arrays. You only ever need numpy arrays to put data into scikit-learn.
+
```

Comment:
Maybe be more explicit:

"""
For instance to train a classifier, all you need is a 2D array `X` for the input variables with the features as columns and samples as rows and a 1D array `y` for the target variable with integer to encode the class membership of each sample in `X`.

To load data as numpy arrays you can use different libraries depending on the original data format:
- `numpy.loadtxt` (link to http://docs.scipy.org/doc/numpy/reference/generated/numpy.loadtxt.html ) to load text files (such as CSV) assuming that all the columns have an homogeneous data type (e.g. all numeric values),
- `scipy.io` (link to http://docs.scipy.org/doc/scipy/reference/io.html ) for common binary formats often used in scientific computing context,
- `scipy.misc.imread` (link to http://docs.scipy.org/doc/scipy/reference/generated/scipy.misc.imread.html#scipy.misc.imread ) (requires the Pillow package) to load pixel intensities data from various image file formats
- `pandas.io` (link http://pandas.pydata.org/pandas-docs/stable/io.html ) to load heterogeneously typed data from various file formats and database protocols that can slice and dice before conversion to numerical features in a numpy array.
"""

Note: if you manage your own numerical data it is recommended to use an optimized file format such as HDF5 to reduce data load times. Various H5Py, PyTables and pandas provide Python interface for reading and writing data with format.